test/utilities/DominoDocumentFragment.js

```diff
@@ -0,0 +1,44 @@
+import assert from 'assert'
+import domino from 'domino'
+
+/**
+ * Gets a document fragment suitable for use in unit tests
+ * Details: https://github.com/fgnass/domino/issues/73#issuecomment-200466430
+ * @param {?String} HTMLString An html string
+ * @return {!domino.impl.DocumentFragment} A domino implementation of a DocumentFragment
+ */
+const documentFragmentFromHTMLString = HTMLString => {
+  const document = domino.createDocument()
+  const template = document.createElement('template')
+  template.innerHTML = HTMLString
+  const fragment = template.content
+  return fragment
+}
+
+describe('DominoDocumentFragment', () => {
```

Comment:
@niedzielski 

I should probably split this test out to its own `TestDominoDocumentFragment.js` file...

Speaking of, could we have tests only run on files in `test` subdirectories if the files themselves start with the word `test`?